apisix/core/request.lua

```diff
@@ -22,6 +22,12 @@
 local lfs = require("lfs")
 local log = require("apisix.core.log")
 local io = require("apisix.core.io")
+local ngx_req
```

Comment:
We don't need a ngx_req as a module variable

Reply:
removed